docs-ref-conceptual/index.md

```diff
@@ -13,49 +13,26 @@ ms.devlang: azurecli
 
 # Azure Command-Line Interface (CLI)
 
-The Azure command-line interface (CLI) is Microsoft's cross-platform command-line experience for managing Azure resources.
-Use it in your browser with [Azure Cloud Shell](/azure/cloud-shell/overview), or [install](install-azure-cli.md) it on macOS,
-Linux, or Windows and run it from the command line.
+The Azure command-line interface (CLI) is Microsoft's cross-platform command-line experience for managing Azure resources. The Azure CLI is easy to learn and the perfect tool for building custom automation that works with Azure resources.
 
-The Azure CLI is easy to get started with, and best used for building automation scripts that work with the Azure Resource Manager.
-Using the Azure CLI, you can create VMs within Azure as easily as typing the following command:
+| <center>![Windows logo](./media/Windows_logo_-_2012.png)<br/>Windows 10</center> | <center>![Ubuntu logo](./media/cof_orange_hex.png)<br/>Ubuntu 16.04+</center> | <center>![macOS logo](./media/Apple_logo_black.png)<br/>macOS</center> | <center>![Azure Cloud Shell logo](./media/cloud-check.png)<br/>Azure Cloud Shell</center> |
```

Comment:
[nit] @kristapratico noticed, it'd be great to make these logos link to their corresponding instructions. I'm not sure if that's a markdown limitation or not.

Reply:
I think that right now we should consider it a limitation of the site style, because if the image is clickable, the text beneath should be as well, and then we start getting styling problems. I think the way the layout is right now works fine as-is.